python/sparkdl/transformers/keras_applications.py

```diff
@@ -70,15 +73,14 @@ def model(self, preprocessed, featurize):
     def inputShape(self):
         pass
 
-    @abstractmethod
-    def numOutputFeatures(self):
+    def testPreprocess(self, inputImage):
         """
-        For testing
+        For testing, the keras preprocess function for the model.
         """
-        pass
+        return self.preprocess(inputImage)
 
     @abstractmethod
-    def testKerasModel(self):
+    def testKerasModel(self, include_top):
```

Comment:
Hm... but then they are not private anyway because of how they are called in tests... ?? Is it weird to make it look private and use it outside the class?